src/Blueprint/Blueprint/Models/PSAssignmentLockSettings.cs

```diff
@@ -0,0 +1,27 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
>Apache [](start = 22, length = 6)

Is this the right license and do we need all files to have it? (I hope not.)

Reply:
Looks like it. I see this in every other project as well. 

---
In reply to: [257120446](https://github.com/Azure/azure-powershell/pull/8537#discussion_r257120446) [](ancestors = 257120446)